src/shillelagh/backends/apsw/vt.py

```diff
@@ -429,10 +462,14 @@ def BestIndexObject(self, index_info: apsw.IndexInfo) -> bool:
         (
             constraints_used,
             index_number,
-            index_name,
+            indexes,
+            orderbys_to_process,
             orderby_consumed,
             estimated_cost,
-        ) = self.BestIndex(constraints, orderbys)
+        ) = self._build_index(constraints, orderbys)
+
+        requested_columns = sorted({column_names[i] for i in index_info.colUsed})
```

Comment:
What sort here , is there any significance to the order as presented?

Reply:
Ok, looks like order doesn't matter since sql lite gives bit mask . Are you sorting just for the test?